### PR TITLE
Fixed REAMDE for persistent connections

### PR DESCRIPTION
**Pull request by crackedmind**

### CODE REVIEW

**Comment by tarcieri** [comment]
Thanks!

**Comment by crackedmind** [comment]
Shame on me. Typo in commit message 😓

File: README.md [hunk 1]
```diff
@@ -271,11 +271,13 @@ reuse the same connection again and again:
 ```ruby
 contents = []
 targets = %w(Hypertext_Transfer_Protocol Git GitHub Linux Hurd)
-HTTP.persistent('http://en.wikipedia.org').do |http|
+HTTP.persistent('http://en.wikipedia.org') do |http|
   targets.each { |target| contents << http.get("/wiki/#{target}") }
 end
 ```
 
+o
```
Comment (SalvatoreT) [inline README.md]: ?

Comment (crackedmind) [inline README.md]: and this :(

Comment (tarcieri) [inline README.md]: Derp

**Comment by tarcieri** [comment]
image

File: README.md (hunk 1 above)
Comment (crackedmind) [inline README.md]: You are to fast for merging :)

Comment (tarcieri) [inline README.md]: I am! Oof!

**Comment by crackedmind** [comment]
Nice picture at all :) Thriller about coder that forgot alphabet, and now some dark organization need to assasin him

**Pull request merged**